packages/app/pages/index.tsx

```diff
@@ -103,6 +159,32 @@ const MembershipCard = (props: {
             leftIcon={<Share1Icon />}
             onClick={() => setShareIsOpen(true)}
           />
+          <Button
+            leftIcon={<Pencil2Icon />}
+            onClick={open}
+          />
+          <Modal
+            isOpen={isOpen}
+            setIsOpen={setIsOpen}
+            title="Edit Membership Type"
+            onConfirm={() => {
+              let upsertData: {
+                [key: string]: string | undefined,
+              } = {};
+              upsertData["redirect_url"] = url;
+              upsertData["contractAddr"] = `${passport.address}`
```

Comment:
why not just use `passport.address` directly ?

Reply:
my bad. was using a different string for dev testing. fixed now